tslint.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "typestrict",
+  "rules": {
+    "no-console": true,
+    "no-debugger": true,
+    "mocha-avoid-only": true,
```

Comment:
I don't find this in the [TSLint docs](https://palantir.github.io/tslint/rules/)?

Reply:
https://github.com/Microsoft/tslint-microsoft-contrib/blob/master/src/mochaAvoidOnlyRule.ts

Reply:
Ok, as far as I get it seems to make sense. 😄

Reply:
Yeah so basically it avoids a situation when you put `it.only` while working on single test case and you commit it (CI would be happy to just run one test case :D) But in our setup code coverage would also found such mistake...